.changeset/many-gifts-strive.md

```diff
@@ -5,7 +5,7 @@
 "section": deprecation
 ---
 
-Deprecate unnecessary exports of container-runtime package
+Many unnecessary exports have been deprecated in the container-runtime package
```

Comment:
Nit: same thought as above - code entity reference should be in code block?
```suggestion
Many unnecessary exports have been deprecated in the `container-runtime` package
```

Reply:
I'm personally in favor of this but I think consensus is to avoid them in headings?